docs/performance/http-cache.md

```diff
@@ -0,0 +1,64 @@
+---
+title: HTTP Cache
+---
+HTTP Cache
+==========
+
+Bolt comes with [Symfony HTTP Cache][] that will act as a simple reverse proxy
+to cache rendered pages on disk for a period of time.
+
+This can provide a performance boost on some pages, as means less work for the
+werb server to do in order to serve the requested page repeatedly in a short
+amount of time.
+
+<p class="note"><strong>Note:</strong> Requests for logged on users will not
+be cached.
+</p>
```

Comment:
Is this "logged on" users, or any user that has a session? Eg. from Members? 

Does it also automatically exclude POST requests for forms?